### PR TITLE
Create edit and delete question endpoints

### LINKED ISSUE #143: Create edit and delete question endpoints
**Is your feature request related to a problem? Please describe.**
We have to provide the admins with the functions to delete and update questions 

**Describe the solution you'd like**
Make two new endpoints, delete questions and edit questions with the underlying service functions and JPA queries(if needed).

### PR DESCRIPTION
**Pull request by Gravewalker666**
## Purpose

The purpose of this PR is to fix #143 

## Goals

- To provide admins with the delete and edit questions functionality

## Approach

- Created a new controller, `QuestionController` under admin endpoints
  - Created two new endpoints edit and delete questions
- Created a new file in the service layer for `QuestionService`
  - Created two new corresponding service functions 

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

## Related PRs
 
#139 

## Test environment
 
- Postman
- Apache Maven 3.6.3
- Java Oracle JDK 8
- macOS Big Sur

### CODE REVIEW

**Review by Piumal1999 — CHANGES_REQUESTED** [review]
Good job @Gravewalker666 
2 minor changes requested

File: src/main/java/org/sefglobal/scholarx/controller/admin/QuestionController.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    }
+
+    @PutMapping()
+    @ResponseStatus(HttpStatus.OK)
+    public List<Question> editQuestions(@RequestBody List<Question> questions) throws ResourceNotFoundException {
+        return questionService.editQuestions(questions);
+    }
+
+    @DeleteMapping("/{id}")
+    @ResponseStatus(HttpStatus.OK)
```
[the remark this answers is not in the document]
Comment (Gravewalker666) [reply src/main/java/org/sefglobal/scholarx/controller/admin/QuestionController.java:27]: On it, thank you for the suggestion

File: src/main/java/org/sefglobal/scholarx/controller/admin/QuestionController.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+@RestController
+@RequestMapping("/api/admin/questions")
+public class QuestionController {
+    private final QuestionService questionService;
+
+    public QuestionController(QuestionService questionService) {
+        this.questionService = questionService;
+    }
+
+    @PutMapping()
```
[the remark this answers is not in the document]
Comment (Gravewalker666) [reply src/main/java/org/sefglobal/scholarx/controller/admin/QuestionController.java]: Resolved with the commit 35fffdf

File: src/main/java/org/sefglobal/scholarx/controller/admin/QuestionController.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Gravewalker666) [reply src/main/java/org/sefglobal/scholarx/controller/admin/QuestionController.java]: Resolved with the commit 35fffdf, Please review =)

**Review by Piumal1999 — APPROVED** [review]
Thank you @Gravewalker666

**Pull request merged**